mqttclients/c/mosquitto_client_extensions/protobuf_handlers/google/protobuf/timestamp.pb-c.c

```diff
@@ -0,0 +1,105 @@
+/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
```

Comment:
Probably best if these get generated based on source. Otherwise, we run the risk of having these out of sync with the definition.

Reply:
Done - generating these files is now a part of the C instructions in the command sample. There are a few other libraries needed for generating these - I only put them in the Command readme since you don't actually need them to run the samples, just to generate the files, but if anyone has other opinions on that I'm fine with putting the installs everywhere